misc/deployments/test4.gno.land/config.toml

```diff
@@ -0,0 +1,247 @@
+
+# Mechanism to connect to the ABCI application: socket | grpc
+abci = "socket"
+
+# Database backend: goleveldb | boltdb
+# * goleveldb (github.com/syndtr/goleveldb - most popular implementation)
+#  - pure go
+#  - stable
+#* boltdb (uses etcd's fork of bolt - go.etcd.io/bbolt)
+#  - EXPERIMENTAL
+#  - may be faster is some use-cases (random reads - indexer)
+#  - use boltdb build tag (go build -tags boltdb)
+db_backend = "goleveldb"
+
+# Database directory
+db_dir = "db"
+
+# If this node is many blocks behind the tip of the chain, FastSync
+# allows them to catchup quickly by downloading blocks in parallel
+# and verifying their commits
+fast_sync = true
+
+# If true, query the ABCI app on connecting to a new peer
+# so the app can decide if we should keep the connection or not
+filter_peers = false
+home = ""
+
+# A custom human readable name for this node
+moniker = "artemis.local" # Change me!
+
+# Path to the JSON file containing the private key to use for node authentication in the p2p protocol
+node_key_file = "secrets/node_key.json"
+
+# Path to the JSON file containing the private key to use as a validator in the consensus protocol
+priv_validator_key_file = "secrets/priv_validator_key.json"
+
+# TCP or UNIX socket address for Tendermint to listen on for
+# connections from an external PrivValidator process
+priv_validator_laddr = ""
+
+# Path to the JSON file containing the last sign state of a validator
+priv_validator_state_file = "secrets/priv_validator_state.json"
+
+# TCP or UNIX socket address for the profiling server to listen on
+prof_laddr = ""
+
+# TCP or UNIX socket address of the ABCI application, 
+# or the name of an ABCI application compiled in with the Tendermint binary
+proxy_app = "tcp://127.0.0.1:26658"
+
+##### consensus configuration options #####
+[consensus]
+
+  # EmptyBlocks mode and possible interval between empty blocks
+  create_empty_blocks = true
+  create_empty_blocks_interval = "0s"
```

Comment:
Can someone remind me of the various configurations we currently have (loop, test3, staging), and had (i.e., gnochess)?

Reply:
you can check them here.

- portal-loop
https://rpc.gno.land/dump_consensus_state

- test3
https://rpc.test3.gno.land/dump_consensus_state

- staging
https://rpc.staging.gno.land/dump_consensus_state